tools/codegen/gen.py

```diff
@@ -643,11 +647,9 @@ def __call__(self, f: NativeFunction) -> Optional[str]:
 // aten::{f.func}
 C10_ALWAYS_INLINE
 {sig.defn(name)} {{
-  static auto op = c10::Dispatcher::singleton()
-    .findSchemaOrThrow("aten::{f.func.name.name}", "{f.func.name.overload_name}")
-    .typed<{dispatcher_sig.type()}>();
   {compute_dk}
-  return op.redispatch(_dk, {', '.join(a.expr for a in dispatcher_exprs)});
+  return at::_ops::{f.func.name.unambiguous_name()}::redispatch(
```

Comment:
What's the justification for moving these calls from the static op to the at::_ops:: namespace?

Reply:
I can back out the change if you want, but fwiw this is a binary size reduction on my machine. 2 MB before and 1.5 MB after using `at::_ops` which makes sense as you are removing all the magic statics code. 

AFAIK this is exactly what the `_ops` API is intended for as well. It makes the generated code simpler, removes a header dependency on the dispatcher internals, centralizes the `static` operator data into one place.

Reply:
Sounds good! Thanks for the clarification.

Reply:
I say it would probably be better to move it to separate PR, as it is not related to "Use per-operator headers" goal, does it?

Reply:
I've split the changes across #70021 and updated the PR description to be explicit about using the `at::_ops` API.